starlite/testing/base/client_base.py

```diff
@@ -50,7 +50,7 @@ def fake_asgi_connection(app: ASGIApp, cookies: Dict[str, str]) -> ASGIConnectio
     )
 
 
-class BaseTestClient:
+class BaseTestClient(Generic[T]):
```

Comment:
This change is not required, additionally its going to break existing uses.

```suggestion
class BaseTestClient:
```

Reply:
@Goldziher but then what would be the type of the `app` argument?

Reply:
my bad